From: Markus Koschany <apo@gambaru.de>
Date: Fri, 27 Feb 2015 00:55:36 +0100
Subject: no e4

Do not import e4 classes because they do not exist in Debian's Eclipse version.
Otherwise this will cause a FTBFS.

Forwarded: not-needed
---
 .../src/org/python/pydev/shared_ui/SharedUiPlugin.java              | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/plugins/org.python.pydev.shared_ui/src/org/python/pydev/shared_ui/SharedUiPlugin.java b/plugins/org.python.pydev.shared_ui/src/org/python/pydev/shared_ui/SharedUiPlugin.java
index 74f4e3a..6f16097 100644
--- a/plugins/org.python.pydev.shared_ui/src/org/python/pydev/shared_ui/SharedUiPlugin.java
+++ b/plugins/org.python.pydev.shared_ui/src/org/python/pydev/shared_ui/SharedUiPlugin.java
@@ -10,8 +10,6 @@ import java.lang.reflect.Field;
 
 import org.eclipse.core.runtime.IStatus;
 import org.eclipse.core.runtime.Status;
-import org.eclipse.e4.ui.css.swt.theme.IThemeEngine;
-import org.eclipse.e4.ui.services.IStylingEngine;
 import org.eclipse.jface.resource.ImageDescriptor;
 import org.eclipse.swt.graphics.Color;
 import org.eclipse.swt.widgets.Control;
@@ -105,7 +103,7 @@ public class SharedUiPlugin extends AbstractUIPlugin {
 
     @SuppressWarnings("restriction")
     public static void setCssId(Object control, String id, boolean applyToChildren) {
-        try {
+/*        try {
             IStylingEngine engine = (IStylingEngine) UIUtils.getActiveWorkbenchWindow().
                     getService(IStylingEngine.class);
             if (engine != null) {
@@ -118,7 +116,7 @@ public class SharedUiPlugin extends AbstractUIPlugin {
             //Ignore: older versions of Eclipse won't have it!
             // e.printStackTrace();
         }
-    }
+*/  }
 
     public static void fixSelectionStatusDialogStatusLineColor(Object dialog, Color color) {
         //TODO: Hack: remove when MessageLine is styleable.
